Automattic/wp-calypso
# Do not ask for backup credentials when authorizing Pressable sites

Related to Automattic/jetpack-backup-team#508.

## Proposed Changes

* Drop the `pressable-nux` flow, which asks for sharing Pressable credentials.

## Why are these changes being made?


* Pressable credentials for Jetpack Backup are already shared by default. There's no need to ask users to share them.

## Testing Instructions



- Create a new Pressable site
- Activate Jetpack at `/wp-admin/plugins.php`
- Connect Jetpack if needed at `/wp-admin/admin.php?page=my-jetpack`
- When Jetpack is connected, click on `Connect your user account` at `/wp-admin/admin.php?page=my-jetpack#/connection`
- In the OAuth authorization screen, the one that says "Your new plan requires a connection to WordPress.com" and there's a green `Approve` button, STOP, don't click on it yet
- Replace the `https://wordpress.com` beginning of the URL by your Calypso Live or Calypso local URL, like `http://calypso.localhost:3000`
- Now click on `Approve`
- You should be redirected back to `/wp-admin/admin.php?page=jetpack#/dashboard` without seeing the Pressable credentials flow

## Pre-merge Checklist
